src/components/CartPriceDetails/CartPriceDetails.jsx

```diff
@@ -0,0 +1,34 @@
+import { useAuthentication } from 'context';
+import React from 'react';
+import "./CartPriceDetails.css";
+
+const CartPriceDetails = () => {
+    const { authState: { cart: cartProvided } } = useAuthentication();
+    const originalPriceSum = cartProvided.reduce((total, everyCartItem) => total += Number(everyCartItem.originalPrice), 0)
+    const discountedPriceSum = cartProvided.reduce((total, everyCartItem) => total += Number(everyCartItem.discountedPrice), 0)
```

Comment:
You can do this in one `reduce` call, return an object which will contain original and discount
```
const  priceSum = cartProvided.reduce((acc, everyCartItem) =>{
return ({...acc, original: Number(everyCartItem.originalPrice) +acc.original, discount:Number(everyCartItem.discountedPrice)+acc.discount})
} , {original:0, discount:0})
```

Reply:
Thanks Jantu for pointing this out !! Will implement it right away . Got a mini issue to be addressed that got solved while using the quantity attribute of every item in the cart